ci/utility/jobs/dev-object-tagging.yml

```diff
@@ -0,0 +1,36 @@
+jobs:
+  - name: rbac-csv-upload-dev
+    max_in_flight: 1
+    serial: true
+    plan:
+      - get: rbac-csv
+        trigger: true
+      - get: aws-common-infrastructure
+      - .: (( inject meta.plan.terraform-output ))
+      - .: (( inject meta.plan.rbac-csv-upload ))
+        config:
+          params:
+            AWS_ROLE_ARN: arn:aws:iam::((aws_account.development)):role/ci
+
+  - name: object-tagging-dev
+    max_in_flight: 1
+    serial: true
+    plan:
+      - get: rbac-csv
+        trigger: true
+        passed:
+          - rbac-csv-upload-dev
+      - get: dataworks-s3-object-tagger
+        trigger: true
+      - put: meta
+        resource: meta
+      - get: aws-common-infrastructure
+      - .: (( inject meta.plan.terraform-output ))
+      - .: (( inject meta.plan.object-tagging-batch ))
+        config:
+          params:
+            BATCH_JOB_DEFINITION: pdm_object_tagger_job
+            AWS_ROLE_ARN: arn:aws:iam::((aws_account.development)):role/ci
+            DATA_S3_PREFIX: "data/uc"
+            CSV_PREFIX: "component/rbac"
+            FILE_NAME: "data_classification.csv"
```

Comment:
can you move all params except `AWS_ROLE_ARN` to `utility/meta.yml` and remove from each environment job

Reply:
May need to be configurable per env. If time shows we don't want to differentiate per environment then it's worth moving to meta.

Reply:
you still can due to the way spruce layers them. These are the default that go in meta, you then can override for specific environments. Currently, this is much harder to maintain when the file name or path changes